### DISCUSSION TITLE
How can I preserve file size when splitting multi page tif to single page tif?

### DISCUSSION THREAD

**Discussion by true-git** [ImageMagick/ImageMagick#2615] [Help] [answered]
I used the command: 

convert -scene 1 input.tif output-&d.tif

The 9 page tif of size 386 KB was converted into 9 single pages of size 1.063 KB each

I want to have 9 single pages of size (386/9=) 43 KB each

**Comment by snibgo** [comment, marked as answer]
"identify -verbose input.tif" will tell you what compression has been applied to each image. You can use the same method in a "-compress" option before you write the output.

**Reply by true-git** [reply]
That worked for me, thanks!
